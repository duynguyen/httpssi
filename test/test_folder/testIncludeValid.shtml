<div>
<!--#include file="included.html" -->
</div>