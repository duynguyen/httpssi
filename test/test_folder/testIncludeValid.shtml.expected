<div>
<h3>Test</h3>
<p>
	I am included.
</p>
</div>